avocado/plugins/runner_nrunner.py

```diff
@@ -258,6 +258,8 @@ def run_suite(self, job, test_suite):
         self.tasks = self._get_all_runtime_tasks(test_suite)
         if test_suite.config.get('nrunner.shuffle'):
             random.shuffle(self.tasks)
+        test_ids = [rt.task.identifier for rt in self.tasks
+                    if rt.task.category == 'test']
```

Comment:
What is rt here? random_task?

Also, unless I'm missing something, this construction gave me the idea that something might be wrong, `random_task.task.category`? `self.tasks` seems to be a list of `tasks`, and a task having another task object inside it sounds weird.

Just trying to understand here.

Reply:
Hum, ok then. My suggestion is just to choose a more descriptive name for this variable, like runtime_tasks.